packages/protocolbeat/src/panel-values/ValuesPanel.tsx

```diff
@@ -95,15 +93,26 @@ function Display({
   chain: string
   blockNumber: number
 }) {
+  const { project } = useParams()
+  if (!project) {
+    throw new Error('Cannot use component outside of project page!')
+  }
   const address = getAddressToCopy(selected)
 
-  const copy = address && canCopy(selected) && !isReadOnly && (
-    <CopyAddShapeCommand
-      chain={chain}
-      address={address}
-      name={selected.name}
-      blockNumber={blockNumber}
-    />
+  const dialog = address && canAddShape(selected) && !isReadOnly && (
```

Comment:
Hide in read only

GIGA nit pick: the animation has a hard step, maybe it's not that hard to remove it and make it smooooooooth from start to finish. Although I don't know how to achieve that